sm_python/src/exportValueStoreRef.cpp

```diff
@@ -119,4 +122,29 @@ void exportExtendibleKeyValueStorePair()
     ///const std::string& getKey() const
     .def("getKey", &ExtendibleKeyValueStorePair::getKey, return_value_policy<copy_const_reference>())
   ;
-}
\ No newline at end of file
+}
+
+void exportLayeredValueStore() {
+  using namespace boost::python;
+  using namespace sm::value_store;
+  
+  class_<LayeredValueStore, bases<ValueStoreRef>>("LayeredValueStore", init<>())
```

Comment:
Shouldn't this be
```suggestion
  class_<LayeredValueStore, bases<ValueStore>>("LayeredValueStore", init<>())
```

I'm surprised that this even compiles :)

Reply:
The reason I used this is because ValueStore itself is not exported
And you are right, while it compiles fine using it does not work, as it complains about mismatch function type on "self"

Reply:
Ah, okay. Thanks for testing. Then we should probably export the ValueStore itself as well.